### ISSUE TITLE
support custom build configurations

### ISSUE THREAD

**Issue by hello-ming** [raycast/extensions#1107]
Raycast version: 1.30.2

I add a new dependency in package.json, but when building the extenstion, it says  **No loader is configured for ".node" file**. 

the new dependency did use .node file, I don't known how to solve this according to the docs. maybe custom esbuild configuration can handle this problem. or if someone known how to do, please leave a message, thx.

**Comment by thomaspaulmann** [comment]
What's the dependency that you want to use?

**Comment by hello-ming** [comment]
chrome-cookies-secure

**Comment by thomaspaulmann** [comment]
We don't support any binary dependencies yet. We might look into it at some point. Any chance you could build your extension without depending on keytar in the meantime?

**Comment by hello-ming** [comment]
Yes, I am already working on plan b.  Use token instead of cookie, it's working now.

**Comment by thomaspaulmann** [comment]
I'm gonna close this issue. It's unlikely that we support custom build settings due to the maintenance effort. We might add support for binary dependencies at some point.
